src/java/org/apache/cassandra/db/compaction/CompactionPick.java

```diff
@@ -138,6 +139,42 @@ public long avgSizeInBytes()
         return avgSizeInBytes;
     }
 
+    public long totSizeInBytes()
+    {
+        return totSizeInBytes;
+    }
+
+    public long parent()
+    {
+        return parent;
+    }
+
+    public ImmutableSet<CompactionSSTable> ssstables()
```

Comment:
I guess it should `sstables()`, not triple `sss`, right? I missed this. Should it be separate patch to fix it?